src/gui.rs

```diff
@@ -302,114 +315,148 @@ impl Application for State {
+        let installation_label = Text::new("Installation:").width(140.into());
+        let installation_client = Radio::new(
+            Installation::Client,
+            "Client",
+            Some(self.installation_type),
+            Interaction::SelectInstallation,
+        );
+        let installation_server = Radio::new(
+            Installation::Server,
+            "Server",
+            Some(self.installation_type),
+            Interaction::SelectInstallation,
+        );
         let installation_row = Row::new()
             .push(installation_label)
-            .push(installation_list)
+            .push(installation_client)
+            .push(installation_server)
             .width(Length::Fill)
-            .align_items(Align::Center)
-            .spacing(5)
+            .align_items(Alignment::Fill)
+            .spacing(50)
             .padding(5);
 
-
-        let minecraft_version_label = Text::new("Minecraft version:").width(Length::Units(140));
+        let minecraft_version_label = Text::new("Minecraft version:").width(140.into());
         let minecraft_version_list = PickList::new(
-            &mut self.minecraft_pick_list,
-            Cow::from_iter(self.minecraft_versions.iter().filter(|v| self.show_snapshots || v.stable).cloned()),
+            Cow::from_iter(
+                self.minecraft_versions
+                    .iter()
+                    .filter(|v| self.show_snapshots || v.stable)
+                    .cloned(),
+            ),
             self.selected_minecraft_version.clone(),
-            Interaction::SelectMcVersion
-        ).width(Length::Fill);
-        let enable_snapshots = Checkbox::new(self.show_snapshots, "Show snapshots", Interaction::EnableSnapshots);
+            Interaction::SelectMcVersion,
+        )
+        .width(200.into());
+        let enable_snapshots = Checkbox::new(
+            self.show_snapshots,
+            "Show snapshots",
+            Interaction::ToggleSnapshots,
+        );
         let mc_row = Row::new()
             .push(minecraft_version_label)
             .push(minecraft_version_list)
+            .push(Space::new(20.into(), 0.into()))
             .push(enable_snapshots)
             .width(Length::Fill)
-            .align_items(Align::Center)
+            .align_items(Alignment::Center)
             .spacing(5)
             .padding(5);
 
-            
-        let loader_version_label = Text::new("Loader version:").width(Length::Units(140));
+        let loader_version_label = Text::new("Loader version:").width(140.into());
         let loader_version_list = PickList::new(
-            &mut self.loader_pick_list,
-            Cow::from_iter(self.loader_versions.iter().filter(|v| self.show_betas || !v.version.contains("beta")).cloned()),
+            Cow::from_iter(
+                self.loader_versions
+                    .iter()
+                    .filter(|v| self.show_betas || !v.version.pre.contains("beta"))
```

Comment:
Similar to how you did before, it probably makes sense to check for `v.version.pre.is_empty()` instead.
Maybe we should in general rename all occurances of `beta` to `pre` or `pre-releases` or siilar to indicate this change.

Reply:
Oops it seems I missed this

Reply:
2afa102175208ad0a3081d4722558ae793074563